lib/hex_web/package.ex

```diff
@@ -157,6 +157,10 @@ defmodule HexWeb.Package do
     |> Enum.any?
   end
 
+  def last_owner?(package) do
+      (length owners package)  == 1
```

Comment:
Good call, in my initial version it was 
```elixir 
length (owners package) == 1
```
but since `==` operator applies before length in this expression I got argument error.